src/server/sams/api/admin/storage_destinations.py

```diff
@@ -0,0 +1,38 @@
+from sams.factory.service import SamsService
+from sams.storage.destinations import destinations
+from superdesk.resource import Resource
+from sams_client.schemas import destinationSchema
+from superdesk.utils import ListCursor
+
+
+class StorageDestinationsResource(Resource):
+    """Resource instance for Storage Destinations
+    """
+
+    endpoint_name = 'destinations'
+    url = 'admin/destinations'
+    item_url = r'regex("[a-zA-Z0-9]+")'
+
+    schema = destinationSchema
+
+    item_methods = ['GET']
+    resource_methods = ['GET']
+
+
+class StorageDestinationsService(SamsService):
+    """Service for storage destinations
+
+    Returns one or all registered storage destinations
+    """
+
+    def get(self, req, **lookup):
+        return ListCursor(list(map(
```

Comment:
Can you add `docstrings` for these two methods, explaining what each method is used for and their restrictions (i.e. `get` returns all `Destinations` regardless of the request parameters or lookup, and `find_one` will only use `_id`)